docs/administration.delete-tenants.md

```diff
@@ -27,4 +27,4 @@ In order to confirm the deletion, you have to first perform the manual deletion
 
 If you choose to decline the deletion, you can do so by clicking on the decline button. If you decline the deletion, the meshTenant will be available again on the workspace control plane.
 
-> If a meshProject was marked for deletion before by a customer, and all meshTenants of the meshProject have been successfully deleted, the meshProject is permanently deleted.
+> If a meshProject was marked for deletion before by a workspace, and all meshTenants of the meshProject have been successfully deleted, the meshProject is permanently deleted.
```

Comment:
d: afaik we've used the term `customer` before to refer to either the customer construct within meshStack, or to an actual person associated to a `customer`. But this doesn't work anymore with the new name, because a workspace is not a person, so I would just use the term `workspace user`.

```suggestion
> If a meshProject was marked for deletion before by a workspace user, and all meshTenants of the meshProject have been successfully deleted, the meshProject is permanently deleted.
```

Reply:
good catch. Mindless search & replace does not work everytime :'D